staging/src/k8s.io/kubectl/pkg/generate/versioned/service.go

```
@@ -210,6 +211,11 @@ func generateService(genericParams map[string]interface{}) (runtime.Object, erro
 			service.Spec.Ports[i].TargetPort = intstr.FromInt(int(port))
 		}
 	}
+	if nodePort, err := strconv.Atoi(params["node-port"]); err == nil {
```

Let's return this error.

---

Fixed as well.